app/views/tutorials/index.html.erb

```diff
@@ -4,7 +4,13 @@
 
 <div class="usa-section main-section">
   <h1><%= @page_title %></h1>
-
+  <p>
+    Tutorials are under construction, coming to you soon! You’ll be able to use them for step-by-step instructions, guiding you through the Defense Personal Property System.
+  </p>
+  <p>
+    If you are currently planning a move and need help today, visit <%= link_to 'your local transportation office', offices_path %> or call the 24-hour Help Desk at <%= link_to '1 (800) 462-2176', 'tel:+18004622176' %>.
+  </p>
+  <!--
```

Comment:
let's just delete this instead of commenting it out, since the design is changing anyway